src/selectors/getChildren.ts

```diff
@@ -74,3 +74,31 @@ export const getChildrenRanked = (state: State, context: Context): Child[] =>
 /** Returns the first visible child of a context. */
 export const firstVisibleChild = (state: State, context: Context) =>
   getChildrenSorted(state, context)[0]
+
+/**
+ * Returns visible children.
+ *
+ * @param state - Redux state.
+ * @param path - Parent's path.
+ * @param simplePath - Parent's simple path.
+ * @param children - Child or ThoughtContext array.
+ * @param showContexts - Parent's context view status.
+ */
+export const filterChildren = (state: State, path: Path, simplePath: SimplePath, children: (Child | ThoughtContext)[], showContexts?: boolean) => {
+  return children.filter(child => {
```

Comment:
In general you want to avoid starting functions with `filter` (or `map`) since it conflates the filtering with the predicate. A standalone predicate function is cleaner and makes it easier to compose filters.

Reply:
@shresthabijay Do you have questions about this?

Reply:
@raineorshine Thanks for pointing it out. I understand it now. Having a separate predicate function makes it easier to compose.